lib/console.js

```diff
@@ -86,7 +86,12 @@ function createWriteErrorHandler(stream) {
   };
 }
 
-function write(ignoreErrors, stream, string, errorhandler) {
+function write(ignoreErrors, stream, string, errorhandler, groupIndent) {
+  if (groupIndent.length !== 0) {
```

Comment:
I would do the latter while I would actually not replace the string again but simply place it like this
```js
if (!ignoreErrors) return stream.write(`${groupIndent}${string}\n`);
```